src/book/03-guides/02-tools/03-webots.mdx

```diff
@@ -133,13 +150,96 @@ sudo apt-get install cmake-curses-gui libprotobuf-dev protobuf-compiler libeigen
    git clone https://github.com/RoboCup-Humanoid-TC/GameController.git
```

Comment:
This doesn't need to be in the home directory I am not sure if it is worth mentioning this